.gitignore

```diff
@@ -60,4 +60,10 @@ xcuserdata
 Tests/Pods
 **/.toDelete
 /test-gh-pages-current/
-doc/Programming-guide/Server-REST-APIs/swagger.json
\ No newline at end of file
+doc/Programming-guide/Server-REST-APIs/swagger.json
+/server/containers/docker/using-compose/kaa-nginx-config/kaa-nginx.conf
+/server/containers/docker/using-compose/kaa-nginx-config/kaa-default.conf
+/logs
+/server/containers/docker/using-compose/kaa-docker-compose.yml
+/server/containers/docker/kaa-node.deb
```

Comment:
this is strange.. is this a side effect of build? We need to cleanup such files, not ignore. The correct location of all build artifacts is in build (target) folder.